qiskit/circuit/classical/expr/__init__.py

```diff
@@ -39,8 +39,8 @@
 
 These objects are mutable and should not be reused in a different location without a copy.
 
-The entry point from general circuit objects to the expression system is by wrapping the object
-in a :class:`Var` node and associating a :class:`~.types.Type` with it.
+The base for dynamic variables is the :class:`Var`, which can be either an arbitrarily typed runtime
+variable, or a wrapper around an old-style :class:`.Clbit` or :class:`.ClassicalRegister`.
```

Comment:
I'm not sure if old-style is the best word here to describe `Clbit` just because it will still be valid and supported for the foreseeable future

Reply:
Removed in 2df8589da0.